Bottom: 8da307aebb8d386f5b764622a5d85fc9f248fa62
Top:    f0373dc0cad41c9ddc737b43e98ceaabe0c77de8
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-01-22 18:45:19 +0800

Refresh of vhost-introduce-helper-to

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 893cad7..3322b82 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -2419,9 +2419,8 @@ static int vhost_read_desc_indices(struct vhost_virtqueue *vq,
 				   u16 num)
 {
 	int ret, ret2;
-	u16 last_avail_idx, last_used_idx, total, copied;
+	u16 last_avail_idx, total;
 	__virtio16 avail_idx;
-	struct vring_used_elem __user *used;
 	int i;
 
 	if (unlikely(vhost_get_avail(vq, avail_idx, &vq->avail->idx))) {
